srcpkgs/libtsm/template

```diff
@@ -1,16 +1,17 @@
 # Template file for 'libtsm'
 pkgname=libtsm
-version=3
-revision=3
-build_style=gnu-configure
+version=4.0.2
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release"
```

Comment:
Rebase and apply this patch:

[0001-fixup-libtsm-update-to-4.0.2-switch-to-Aetf-s-fork.txt](https://github.com/void-linux/void-packages/files/10326860/0001-fixup-libtsm-update-to-4.0.2-switch-to-Aetf-s-fork.txt)